### PR TITLE
Add pkg.outputPath

### PR DESCRIPTION
**Pull request by Symbitic**
This adds a `pkg.outputPath` setting inside `package.json`. When given, output files will always be placed in this directory. It's identical to passing `--out-path` to the command-line.

I made this because it seems like the sort of thing people would want to specify in a configuration file, instead of needing to pass it to the command-line every time.
I also updated the README to explain how to use it, and I also added instructions about `pkg.targets`.

I'm relatively new to Git (and especially GitHub pull requests), so let me know if I did anything wrong.

### CODE REVIEW

**Comment by rauchg** [comment]
Would you mind adding tests @Symbitic ? Then we can merge it

**Comment by Symbitic** [comment]
I submitted a test. Do you know when it can be merged?

File: README.md [hunk 1]
```diff
@@ -61,29 +61,35 @@ are taken from current platform/Node.js. By default targets are
...
-    "assets": "views/**/*"
+    "assets": "views/**/*",
+    "targets": [ "node4-linux-armv6" ],
+    "outputPath: "dist"
   }
 ```
-You may also specify arrays of globs:
-```
-    "assets": [ "assets/**/*", "images/**/*" ]
-```
```
Comment (robertsLando) [inline README.md:84]: Please restore this lines

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Comment by robertsLando** [comment]
@leerob this can be merged once tests pass

**Comment by leerob** [comment]
The satisfaction merging this multiple year old PR

**Review by leerob — APPROVED** [review]

**Pull request merged**